<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WFornecedorRequisitar</class>
 <widget class="QWidget" name="WFornecedorRequisitar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>761</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>761</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>17</number>
   </property>
   <property name="leftMargin">
    <number>13</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>13</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignLeft">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>35</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <widget class="QPushButton" name="pbVoltar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="shortcut">
       <string>Ctrl+B</string>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSplitter" name="splitter_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>118</width>
       <height>27</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="outTitulo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>118</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>118</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>Requisição</string>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSplitter" name="splitter_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fornecedor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outFornecedor">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="tmp">
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSplitter" name="splitter_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Produto</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outProduto">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="tmp_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSplitter" name="splitter_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="inQuantidade">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="errQuantidade">
      <property name="text">
       <string>Por favor, informe a quantidade.</string>
      </property>
      <property name="err" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSplitter" name="splitter_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Previsão</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="inPrevisao">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>29</height>
       </size>
      </property>
      <property name="displayFormat">
       <string>dd/MM/yyyy</string>
      </property>
     </widget>
     <widget class="QLabel" name="errPrevisao">
      <property name="text">
       <string>Por favor, informe a previsão.</string>
      </property>
      <property name="err" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSplitter" name="splitter_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>35</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <widget class="QPushButton" name="pbConfirmar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>147</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>147</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="shortcut">
       <string>Ctrl+O</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCancelar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>147</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>147</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="shortcut">
       <string>Ctrl+A</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>outFornecedor</tabstop>
  <tabstop>outProduto</tabstop>
  <tabstop>inQuantidade</tabstop>
  <tabstop>inPrevisao</tabstop>
  <tabstop>pbConfirmar</tabstop>
  <tabstop>pbCancelar</tabstop>
  <tabstop>pbVoltar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>inQuantidade</sender>
   <signal>editingFinished()</signal>
   <receiver>WFornecedorRequisitar</receiver>
   <slot>checarQuantidade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inPrevisao</sender>
   <signal>editingFinished()</signal>
   <receiver>WFornecedorRequisitar</receiver>
   <slot>checarPrevisao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbConfirmar</sender>
   <signal>clicked()</signal>
   <receiver>WFornecedorRequisitar</receiver>
   <slot>adicionar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbConfirmar</sender>
   <signal>clicked()</signal>
   <receiver>WFornecedorRequisitar</receiver>
   <slot>voltar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancelar</sender>
   <signal>clicked()</signal>
   <receiver>WFornecedorRequisitar</receiver>
   <slot>voltar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbVoltar</sender>
   <signal>clicked()</signal>
   <receiver>WFornecedorRequisitar</receiver>
   <slot>voltar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>adicionar()</slot>
  <slot>voltar()</slot>
  <slot>checarQuantidade()</slot>
  <slot>checarPrevisao()</slot>
 </slots>
</ui>
